app/controllers/ConfirmationController.scala

```diff
@@ -59,21 +59,24 @@ class ConfirmationController @Inject() (
           pf,
           rp.value.toDouble
         )
-        if (grant.isEligible) {
-          Ok(view(grant, appConfig.calculatorVersion))
-        } else {
-          Ok(view(grant, appConfig.calculatorVersion))
-          //Redirect(routes.YouAreNotEligibleController.onPageLoad())
-        }
-      case _                                                =>
+        Ok(view(grant, appConfig.calculatorVersion))
+
+      case _ =>
         Logger.warn("expected data is missing from userAnswers, redirecting user to start page")
         Redirect(routes.ClaimPeriodController.onPageLoad())
     }
 
   }
 
-  private def periodsWithHours(periods: List[Period], hours: Seq[UsualAndActualHours]) =
-    periods.zip(hours).map { x =>
+  private def periodsWithHours(periods: List[Period], hours: Seq[UsualAndActualHours]) = {
```

Comment:
make that hours a List[A]
and then do
if (hours.isEmpty)
 // something
else
 // something